### ISSUE TITLE
Metadata mappings for CORE OAI-PMH

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/purl#866]
Create metadata mappings so that we can provide access to the ETD materials (those submitted via the ETD application, not via H2 or other means) and items in the Stanford University Open Access Articles collection (druid:nk071jz2236) via OAI-PMH to the CORE service (http://core.ac.uk).

See [Best practice for CORE harvesting of data providers-v3](https://docs.google.com/document/d/1sc8RSAhJT4kmYxUKgKvNbPSegejosr83VIqOo6CR7mY/edit#heading=h.e40mgc5opd98), particularly section 3.2 on metadata configuration.

**Comment by amyehodge** [comment]
Proposal for this work can be found in #795.

**Comment by arcadiafalcone** [comment]
@amyehodge Is there a desire to expand CORE harvesting to non-ETD items in future?

**Comment by amyehodge** [comment]
Yes. Definitely to the open access and other research publications, but potentially to other text content like non-ETD theses and capstones, grey literature, technical reports, etc. But I'd say OA publications would be top of that expanded list.

**Comment by arcadiafalcone** [comment]
Recommended CORE meta-tags:

* Title
* Author
* Publication date
* Journal title (or conference title) [not relevant for ETDs]
* Publisher, that is, the institution (for theses and dissertations)

Title and author are already extracted for schema.org tags. Publication date will always be present in the descriptive metadata (an ETD record should include only two dates, publication and copyright, so the correct date may be easily identified). Publisher would be Stanford University for all objects.

**Comment by arcadiafalcone** [comment]
CORE metadata schemas:

* Dublin Core/Extended Dublin Core (minimal)
* OpenAIRE (supported)
* RIOXX (recommended)

If the focus in on ETDs (metadata automatically generated from a template) and open-access articles deposited via H2 (user-created metadata restricted by interface), a relatively small and constant set of metadata fields need to be mapped.

**Comment by lwrubel** [comment]
Noting here that our current Dublin Core mapping is suboptimal, so we should consider whether it is worth putting effort into improving that or if it would be more effective to use one of the other supported/recommended metadata schemas.

**Comment by arcadiafalcone** [comment]
@amyehodge @lwrubel Do we already have a sense of which of the three CORE metadata schemas we want to use?

**Comment by amyehodge** [comment]
@arcadiafalcone This topic came up in a meeting with Tom, Vivian, and Rochelle last week and was noted as a point that needs discussion, and, no we do not already have a sense of which of the three options we would want to use.

**Comment by arcadiafalcone** [comment]
@amyehodge What information do we need to make the decision? I can do analysis from the ease/completeness-of-mapping standpoint, but am less familiar with what other concerns might be.

**Comment by amyehodge** [comment]
@arcadiafalcone I think that information would be really helpful if you'd like to start on that. There is a meeting tomorrow where I might be able to at least start trying to suss this out. But ease/completeness-of-mapping will definitely factor in.
